### DISCUSSION TITLE
useFetcher load doubt

### DISCUSSION THREAD

**Discussion by WildEgo** [remix-run/remix#2453] [Q&A] [answered]
Hey guys, picker up the framework not long ago and started playing with it, finally found the first part where I'd need to use ``useFetcher``, I tried to get the loader data from a certain route but it returns me the loader function of the root as opposed to the one I asked for.

```tsx
  const fetcher = useFetcher();

  const loadMore = useCallback(() => {
    if (!nextPage) return;
    fetcher.load(nextPage);
  }, [nextPage, fetcher]);
  ```
  
  It's pretty simple and straight forward, I might be doing something wrong and if so I'd hope someone could explain it to me.
  
  Thanks in advance,
  Luis Bizarro.

**Comment by kiliman** [comment, marked as answer]
What is `nextPage`? 

If you're trying to load the *index* route, then the url should include `?index`. This is because the URL `/` matches ***both*** the *root* route and the *routes/index* route. Remix uses `?index` to disambiguate them.

**Reply by WildEgo** [reply]
Thank you mate I for some reason didn't read that, I forgot to reply before but this is the answer.
